school/person.rb

```diff
@@ -19,6 +20,10 @@ def of_age?
 
   public
 
+  def add_rental(date, book)
+    Rental.new(date, book)
+  end
```

Comment:
- [ ] Great job so far. Please, like the previous reviewer noted, The person itself should be sent as a parameter to create the new rental, achieving the 'has-many' association. Kindly include `self `like the previous review précised.
ie
```ruby
 def add_rental(book, date)
    Rental.new(date, book, self)
  end
```